### PR TITLE
DOC Fixing EX01 - added examples

### PR DESCRIPTION
**Pull request by DeaMariaLeon**
Towards #37875

I couldn't locally build the html files for `Series.SeriesGroupBy.is_monotonic_increasing` and `Series.SeriesGroupBy.is_monotonic_decreasing`. Followed https://github.com/noatamir/pyladies-berlin-sprints/issues/13 to split the docstrings from Series - but I'm not sure if adding #GH13 to the docstring is correct

Couldn't use a variable like {klass} for `item()` to avoid hardcoding `Series` or `Index`.

### CODE REVIEW

File: pandas/core/groupby/generic.py [hunk 1]
```diff
@@ -1170,13 +1170,43 @@ def cov(
         return result
 
     @property
-    @doc(Series.is_monotonic_increasing.__doc__)
     def is_monotonic_increasing(self) -> Series:
+        # GH13
```
Comment (mroeschke) [inline pandas/core/groupby/generic.py:1174]: Yeah we don't need this (and below)

Comment (DeaMariaLeon) [reply pandas/core/groupby/generic.py:1174]: Thanks, I'll remove them.

**Comment by DeaMariaLeon** [comment]
I can't tell (for sure) if failing tests are related to my changes

File: pandas/core/base.py [hunk 2]
```diff
@@ -989,6 +1013,12 @@ def is_monotonic_decreasing(self) -> bool:
         Returns
         -------
         bool
+
+        Examples
+        --------
+        >>> s = pd.Series([3, 2, 2, 1])
+        >>> s.is_monotonic_decreasing
```
Comment (MarcoGorelli) [inline pandas/core/base.py:1020]: for this and the above, could we add an example for when the return value is `False`?

In general, for functions which return `-> bool`, it's probably good to have a simple example of both cases (something quick and simple is fine)

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]
Nice work!

I _think_ the changes are unrelated - I've just made a minor request, if you make that and then also fetch and merge upstream/main then I'd like to think that CI will be fixed

File: pandas/core/base.py [hunk 3]
```diff
@@ -965,6 +977,12 @@ def is_unique(self) -> bool:
         Returns
         -------
         bool
+
+        Examples
+        --------
+        >>> s = pd.Series([1, 2, 3, 1])
+        >>> s.is_unique
+        False
```
Comment (MarcoGorelli) [inline pandas/core/base.py:985]: this one also returns `bool` ;) rest looks good though!

File: pandas/core/base.py (hunk 2 above)
Comment (DeaMariaLeon) [reply pandas/core/base.py:1024]: Done, thanks for the instructions.

File: pandas/core/base.py (hunk 3 above)
Comment (DeaMariaLeon) [reply pandas/core/base.py:989]: I should have seen it :) .. thanks

File: pandas/core/groupby/generic.py [hunk 4]
```diff
@@ -1170,13 +1170,41 @@ def cov(
...
+        Parrot     True
+        dtype: bool
+        """
         return self.apply(lambda ser: ser.is_monotonic_increasing)
 
     @property
-    @doc(Series.is_monotonic_decreasing.__doc__)
     def is_monotonic_decreasing(self) -> Series:
+        """
+        Return boolean if values in the group are monotonically decreasing.
```
Comment (MarcoGorelli) [inline pandas/core/groupby/generic.py:1194]: so sorry, I missed this the first time round

it doesn't return a boolean, but a Series of booleans

File: pandas/core/groupby/generic.py [hunk 5]
```diff
@@ -1170,13 +1170,41 @@ def cov(
...
+
+        Returns
+        -------
+        bool
```
Comment (MarcoGorelli) [inline pandas/core/groupby/generic.py:1198]: Series

File: pandas/core/groupby/generic.py [hunk 6]
```diff
@@ -1170,13 +1170,41 @@ def cov(
...
 
     @property
-    @doc(Series.is_monotonic_increasing.__doc__)
     def is_monotonic_increasing(self) -> Series:
+        """
+        Return boolean if values in the group are monotonically increasing.
+
+        Returns
+        -------
+        bool
```
Comment (MarcoGorelli) [inline pandas/core/groupby/generic.py:1179]: same comment

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]
new examples look good, thanks! I just missed one last (I promise!) comment last time

File: pandas/core/groupby/generic.py (hunk 4 above)
Comment (DeaMariaLeon) [reply pandas/core/groupby/generic.py]: My fault 🫣

File: pandas/core/groupby/generic.py (hunk 5 above)
Comment (DeaMariaLeon) [reply pandas/core/groupby/generic.py]: Changed

File: pandas/core/groupby/generic.py (hunk 6 above)
Comment (DeaMariaLeon) [reply pandas/core/groupby/generic.py]: Done

**Review by MarcoGorelli — APPROVED** [review]
cool, let's get this in - thanks @DeaMariaLeon !

**Pull request merged**